presto-docs/src/main/sphinx/release/release-0.148.rst

```diff
@@ -13,37 +13,97 @@ General Changes
   connectors that expose node-partitioned data.
 * Add support for ``REVOKE`` permission syntax.
-* Warn if Presto server is not using G1 garbage collector.
-* Improve planning of co-partitioned JOIN and UNION.
-* Improve planning of aggregations over partitioned data.
 * Add support for colocated unions.
-* Use HTTPS in JDBC driver when using port 443.
-* Increase default value for ``query.initial-hash-partitions`` to ``100``.
 * Add :func:`element_at` function for map type.
 * Add :func:`split_to_map` function.
+* Add :func:`zip` function.
 * Add ``ROW`` syntax for constructing row types.
+* Add support for `SMALLINT` and `TINYINT` types.
+* Add :func:`map_union` aggregation function.
+* Add re2j-td as backend for regular expression functions, this can be enabled via
+  the ``regex-library`` config.
+* Add ``tables`` system table in raptor connector.
+* Add support for non-equi outer joins.
 * Reduce initial memory usage of :func:`array_agg` function.
+* Improve planning of co-partitioned JOIN and UNION.
+* Improve planning of aggregations over partitioned data.
+* Improve the performance of :func:`array_sort` function.
+* Improve aggregation performance.
+* Increase default value for ``query.initial-hash-partitions`` to ``100``.
 * Change default value of ``query.max-memory-per-node`` to ``10%`` of the Java heap.
+* Change default ``task.max-worker-threads`` to ``2`` times the number of cores.
+* Use HTTPS in JDBC driver when using port 443.
+* Warn if Presto server is not using G1 garbage collector.
+
+Verifier Changes
+----------------
+
+* Add ``skip-cpu-check-regex`` config property which can be used to skip the CPU
+  time comparison for queries that match the given regex.
+* Add ``check-cpu`` config property which can be used to disable CPU time comparison.
 
 Hive Changes
 ------------
 
 * Fix ``NoClassDefFoundError`` for ``KMSClientProvider`` in HDFS client.
 * Fix creating tables on S3 in an empty database.
 * Implement ``REVOKE`` permission syntax.
-
+* Add support for `SMALLINT' and `TINYINT`
+* Support ``DELETE`` from unpartitioned tables.
+* Add support for Kerberos authentication when talking to Hive/HDFS.
+* Push down filters for columns of type ``DECIMAL``.
 
 Cassandra Changes
 -----------------
 
 * Allow configuring load balancing policy and no host available retry.
+* Add support for ``varchar(n)``.
 
 Kafka Changes
 -------------
 
 * Update to Kafka client 0.8.2.2. This enables support for LZ4 data.
 
+MySql & Postgres Changes
+------------------------
+
+* Push down predicates for ``VARCHAR``, ``DATE``, ``TIME``, and ``TIMESTAMP`` types.
+
+Example Http Changes
+--------------------
+
+* Add support for ``varchar(n)``.
+
+Redis Changes
+-------------
+
+* Add support for ``varchar(n)``.
+
+TPCH Changes
+------------
+
+* Add support for ``varchar(n)``.
+
+Local File Changes
+------------------
+
+* Add support for ``varchar(n)``.
+
+Mongodb Changes
```

Comment:
MongoDB is written as "MongoDB"

See the connector documentation for the proper way to capitalize and spell each connector's name